docs/resources/index.md

```diff
@@ -1,25 +1,83 @@
 # Resources for Organizers
 
-The eScience Institute supports Community Partners in planning each hackweek. We invite organizers to join a series of planning sessions to prepare for an event. Not everyone is expected to attend every session, and your level of commitment will depend on your chosen role.
+The eScience Institute supports Community Partners in planning each hackweek. We invite organizers to participate in video and live training sessions tailored to each of the Community Partner roles. Upon completion of the training, organizer profiles will be posted on the hackweek website along with badge(s) representing their roles in the event. 
 
-## Planning Sessions
+::::{grid}
+:gutter: 1
 
-We host a series of 8 to 10 weekly planning sessions on zoom during the two months prior to a hackweek. These sessions include:
+:::{grid-item-card} [Tutorial Lead](#tutorial-lead)
```

Comment:
Currently these links go to the meeting schedule info on this page - should we have the title or badge link back to the actual role page instead?

Reply:
I saw these as navigation tools within the page, and wonder if it will be confusing if they link back to the roles page?

For now I'll suggest we have an option to link back to the roles within the text describing the schedules. We can revisit this later if it doesn't seem to flow well.